EIPS/eip-draft_multiple_contenthash_records.md

```diff
@@ -0,0 +1,79 @@
+
+The proposed implementation was chosen because it is simple to implement and supports all important requested features. However, it doesn't support multiple records for the same type and priority order, as they aren't so important and are harder to implement properly. To implement them, a different way of implementation would need to be used.
+
+## Backwards Compatibility
+<!--All EIPs that introduce backwards incompatibilities must include a section describing these incompatibilities and their severity. The EIP must explain how the author proposes to deal with these incompatibilities. EIP submissions without a sufficient backwards compatibility treatise may be rejected outright.-->
+
+The EIP is backwards-compatible with EIP 1577, the only difference is additional overloaded methods. Old applications will still be able to function correctly, as they will receive the default `contenthash` record.
+
+## Test Cases
+<!--Test cases for an implementation are mandatory for EIPs that are affecting consensus changes. Other EIPs can choose to include links to test cases if applicable.-->
+
+Automatic Truffle tests have also been written along with new contract code. However, additional manual tests are also needed.
+
+## Implementation
+<!--The implementations must be completed before any EIP is given status "Final", but it need not be completed before the EIP is accepted. While there is merit to the approach of reaching consensus on the specification and rationale before writing code, the principle of "rough consensus and running code" is still useful when it comes to resolving many discussions of API details.-->
+
+Implementation of the smart contract, as well as automatic tests, are written in [`ensdomains/resolvers#30`](https://github.com/ensdomains/resolvers/pull/30).
+
+## Security Considerations
+<!--All EIPs must contain a section that discusses the security implications/considerations relevant to the proposed change. Include information that might be important for security discussions, surfaces risks and can be used throughout the life cycle of the proposal. E.g. include security-relevant design decisions, concerns, important discussions, implementation-specific guidance and pitfalls, an outline of threats and risks and how they are being addressed. EIP submissions missing the "Security Considerations" section will be rejected. An EIP cannot proceed to status "Final" without a Security Considerations discussion deemed sufficient by the reviewers.-->
+
+There SHOULD NOT be any security problems with this EIP. However, it SHOULD still be audited to check for any possible issues with using `mapping` and `bytes` for storage, and overloading methods, as well as other implementation-related features.
```

Comment:
EIPs should be written targeting a future reader (5 to 10 years) long after the EIP is final.  If this EIP is not audited before then, then it probably never will be audited so commenting that someone should audit this EIP should be removed.  It may be valuable warn implementers about security related gotchas if they exist in this section.